<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <item row="1" column="0" colspan="3">
     <widget class="QDial" name="dial">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="notchTarget">
       <double>10.000000000000000</double>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>simple alarm</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="displayFormat">
       <string>yyyy/MM/dd H:mm</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLCDNumber" name="lcdNumber_2">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>dial</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
